transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpHeaders.java

```diff
@@ -37,6 +37,7 @@
     String HEADER_X_ORIGIN = "X-Origin";
     String HEADER_X_CREATE_ENCODING = "X-Create-Encoding";
     String HEADER_X_ACCEPT_COMMANDS = "X-Accept-Commands";
+    String HEADER_X_FORWARDED_FOR = "X-Forwarded-For";
```

Comment:
Good catch @jitsni.

Specifically [RFC 7239](http://tools.ietf.org/html/rfc7239) seems dedicated to this, using `Forwarded` header and not the defacto, non-standard `X-Forwarded-For` header.

Suggest we should be following the specification here, now that it has been explicitly added as an RFC.